vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb

```diff
@@ -58,6 +60,18 @@ def actual_duration_mins
       range.duration_mins
     end
 
+    def problem_description
+      if entry_at.blank?
+        text(:missing_entry)
+      elsif exit_at.blank?
+        text(:missing_exit)
+      end
+    end
+
+    def translation_scope
+      "secure_rooms/occupancy"
```

Comment:
If we're putting the other model messages under `activerecord.models`, we should probably do the same here.